docs/fabrication/cnc/3d-printer.md

```diff
@@ -0,0 +1,71 @@
+# 3D Printers
+![Picture of Ultimaker 3D Printer](https://cdn.discordapp.com/attachments/983922080879869962/1135469515489148998/Ultimaker_History_-_6_Ultimaker_2.png)
+## What is a 3D Printer?
+3D printing is always considered a form of additive manufacturing. A 3D printer creates a 3D object layer by layer only where material is needed. In doing this, you can make custom parts for exactly what you want. If you can design it, a 3D printer can make it. :D
+## Why use a 3D Printer?
+Using a 3D printer versus something like a Laser Cutter has its advantages. With a Laser Cutter, you are limited to the piece of material that you put in. But, if you are creating multiple flat objects, a Laser Cutter can do it faster, and more efficiently. Typically, when using a Laser Cutter, you also need a air filtration system, but with some materials in 3D Printing, you do not need a filtration system.
+
+## Types of 3D Printers
+### FDM (Fused Deposition Modeling)
+FDM or Fused Deposition Modeling is the most common form of 3D printing thanks to its ease of use and cost-effectiveness. FDM printing uses a strand of plastic which is melted and pushed through a nozzle to form your model. 
+### SLA (Stereolithography)
+SLA or Stereolithography is a resin based form of 3D printing. This uses a UV based resin and a screen to cure the resin in certain spots. Resin printing tends to have great detail and quality, but has toxic fumes, and must be cured in a light box after printing. 
+### SLS (Selective Laser Sintering)
+SLS or Selective Laser Sintering is a very expensive form of 3D printing which uses lasers and a powder which is melted to form your 3D model. With SLS, you can print with extremely advanced materials, and even metal. The cleanup tends to be a little messy, and due to its price is usually only used in enterprise applications. 
+
+## Common Types of Filament for FDM
```

Comment:
it might be better to simply create a table for like each property of the material